<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileFilterWindow</class>
 <widget class="QMainWindow" name="FileFilterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBoxC">
      <property name="text">
       <string>*.c</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBoxJava">
      <property name="text">
       <string>*.java</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBoxCpp">
      <property name="text">
       <string>*.cpp</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBoxCs">
      <property name="text">
       <string>*.cs</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxH">
      <property name="text">
       <string>*.h</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkBoxTxt">
      <property name="text">
       <string>*.txt</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBoxOther">
      <property name="text">
       <string>other</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTextEdit" name="textEditFilterText"/>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButtonOk">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>229</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
